### ISSUE TITLE
240 cookies, only 2 FB bots?

### ISSUE THREAD

**Issue by Cyanox62** [davidmann4/ogar-feeder-bot#474]
I have 240 cookies and a working FB bot script, but i only get around 2-3 FB bots ingame, all the rest are normal, any ideas?

**Comment by wartoz** [comment]
me too often i got 400+ working fb cookies and updated fb script i don't know how to fix

**Comment by ravyn120** [comment]
Do you use the free fb bot script and your cookies might be banned

**Comment by hassanila** [comment]
No.. cookies are not banned.. thats not the prob..   the prob is that you would need to edit agar.io client to use token only when bot is actually connected and not before.              What is happening is that the bots get the token and try to connect trough the proxy.. if the proxy fails = token is lost = no fb bot.

**Comment by ravyn120** [comment]
Ohh okay that seems pretty easy to do

**Comment by wartoz** [comment]
have you a fix for this hassanila97 ?

**Comment by wartoz** [comment]
agarbot.ovh have it i think

**Comment by hassanila** [comment]
Yes, I've fixed it in my code.. but as said my code is completely rewritten and doesn't even use agar.io-client.. and it's private  NO SHARE :D   Its easy to implement if you can even 1% JS

**Comment by ravyn120** [comment]
@Wartoz i'll make it when I have time

**Comment by wartoz** [comment]
thanks ravyn120

**Comment by Nick8fr32** [comment]
I dont get this u have 100+ cookies and get Like 2 fb bots when i use 10 cookies i get Like 8-10 fb bots?

**Comment by Cyanox62** [comment]
@ravyn120 when you make it can yoi post it here?

**Comment by Cyanox62** [comment]
@hassanila97 if I get premium proxies that all work, then because no proxies would fail all tokens should work right?

**Comment by hassanila** [comment]
yes

On 24 April 2016 at 23:37, Rassyp notifications@github.com wrote:

> @hassanila97 https://github.com/hassanila97 if I get premium proxies
> that all work, then because no proxies would fail all tokens should work
> right?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> https://github.com/davidmann4/ogar-feeder-bot/issues/474#issuecomment-214041603

## 

_Med vänliga hälsningar_
_Hassan_

**Comment by Cyanox62** [comment]
@hassanila97 what file would I modify to make the tokens connect only to bote who are connected?

**Comment by hassanila** [comment]
agario-client.js

**Comment by Cyanox62** [comment]
@hassanila97 you know how when you start the bots it checks and counts the amount of tokens, should I change it so it does that after it connects the bots? Or am I supposed to do something other than that

**Comment by Newpredator** [comment]
Agarbot.ovh no is fixed. It give the accounts of users. look at the extension at back.js :/

**Comment by DeepSnowNeeL** [comment]
Yes we do, read our ToS. :)

**Comment by sejo153** [comment]
@hassanila97 If you want i give you 30 cookies to make it for me. Thats all i have btw

**Comment by CupidStunts** [comment]
@DeepSnowNeeL so ovh steals FB cookies? WTF

**Comment by hassanila** [comment]
Yes, most extensions and userscript also do, you need to edit the code before actually installing any extension :P

**Comment by CupidStunts** [comment]
@hassanila97 thats why i always use a G+ account haha

**Comment by sejo153** [comment]
@hassanila97 Answer my question at least? xD

**Comment by SanjaySasimandir** [comment]
@Rassyp Hey dude, do you know if your cookies are good? If they are it might be the proxies. the cookies need proxies to connect to the game. So it is important to use good proxies from trusted sites. Use either http://proxybag.blogspot.com/ or http://www.vipsocks24.net/ both. They have very good proxies and mostt of them work. Hope this fixes your problem.  :D

**Comment by Cyanox62** [comment]
Ive used those before, still only 2-3 FB bots. All of the cookies are good though :/

**Comment by ghost** [comment]
@hassanila97 Please share it to me on skype, I will praise youu

**Comment by ravyn120** [comment]
It's simple set auth token to null and emit when the bot is connect to server the add token and re try like 5 times

**Comment by sejo153** [comment]
@ravyn120 Can you help me about that, i have tried but not work

**Comment by ghost** [comment]
@sejo153 Me too

**Comment by Cyanox62** [comment]
@ravyn120 Could you help me with this? I would appreciate it a lot :)
Add my skype: zpasswordz

**Comment by anonymousxxx02** [comment]
@Rassyp can you give me 20 cookie to test my script ???????
